app/assets/stylesheets/pages/_contact.scss

```diff
@@ -1,4 +1,77 @@
+// .collapsing for custom transition with bootstrap
+.collapsing {
+  -webkit-transition-duration: 3.35s;
+  transition-duration: 3.35s;
+  min-height: 120px;
+}
+
 .biographie {
   margin : 40px;
   margin-left: 0px;
+
+   p {
+    margin: 0px;
+   }
+}
+
+.contact-wrapper {
+  display: flex;
+  justify-content: space-between;
+}
+
+// EXHIBITIONS BOXES
+
+.exhibitions-box {
+ width: 500px;
+ box-shadow: $box-shadow;
+ height: fit-content;
+
+  h2 {
+    text-align: center;
+    font-weight: bold;
```

Comment:
Maybe try add a bit of padding to the titles like "Expositions Personnelles", top and bottom ?